server/src/test/java/org/elasticsearch/index/mapper/DocumentMapperTests.java

```diff
@@ -284,8 +295,59 @@ public void testEmptyDocumentMapper() {
         assertNotNull(documentMapper.idFieldMapper());
         assertNotNull(documentMapper.sourceMapper());
         assertNotNull(documentMapper.IndexFieldMapper());
-        assertEquals(10, documentMapper.mappers().getMapping().getMetadataMappersMap().size());
-        assertEquals(10, documentMapper.mappers().getMatchingFieldNames("*").size());
```

Comment:
These assertions failed in my PR but were hard to read. So I made them easier to read while fixing them.